### PR TITLE
Wiki id added and transcription

### PR DESCRIPTION
**Pull request by sologebre**

### CODE REVIEW

File: 2001-3000/LOC2472Dambac.xml [hunk 1]
```diff
@@ -54,14 +54,18 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPlace>
-            <place type="awragga">
-               <placeName>Dämbäḉa</placeName>
-            </place>
+            <place type="awragga" sameAs="wd:Q3022069">
+               <placeName xml:lang="gez" xml:id="n1">ደምበጫ፡</placeName>
+               
+               <placeName xml:lang="gez" corresp="#n1" type="normalized">c</placeName>
```
Comment (thea-m) [inline 2001-3000/LOC2472Dambac.xml]: the transliteration is missing :)

File: 4001-5000/LOC4189Kutaba.xml [hunk 2]
```diff
@@ -10,7 +10,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title>Kutabär</title>
+            <title>Kutabar</title>
```
Comment (thea-m) [inline 4001-5000/LOC4189Kutaba.xml]: Kutābar

File: 4001-5000/LOC4189Kutaba.xml [hunk 3]
```diff
@@ -54,13 +54,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPlace>
-            <place type="region">
-               <placeName>Kutabär</placeName>
+            <place type="region" sameAs="wd:Q3200672">
+               <placeName xml:lang="gez" xml:id="n1">ኩታበር፡</placeName>
+               <placeName xml:lang="gez" corresp="#n1" type="normalized">Kutabar</placeName>
```
Comment (thea-m) [inline 4001-5000/LOC4189Kutaba.xml]: Kutābar

File: 4001-5000/LOC4578Marhab.xml [hunk 4]
```diff
@@ -54,13 +54,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+         <change who="SG" when="2020-02-19">added Wiki id, transcription</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPlace>
-            <place type="region">
-               <placeName>Märḥabete</placeName>
+            <place type="region" sameAs="wd:Q3219955">
+               <placeName xml:lang="gez" xml:id="n1">መርሓቤቴc</placeName>
```
Comment (thea-m) [inline 4001-5000/LOC4578Marhab.xml]: I think the c after the fidal does not belong there

File: 5001-6000/LOC5542Sanga.xml [hunk 5]
```diff
@@ -54,13 +54,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPlace>
-            <place>
-               <placeName>Sanga</placeName>
+            <place type="warada" sameAs="wd:Q3472289">
+               <placeName xml:lang="gez" xml:id="n1">ሳንጃ፡</placeName>
+               <placeName xml:lang="gez" corresp="#n1" type="normalized">Sāngā</placeName>
```
Comment (thea-m) [inline 5001-6000/LOC5542Sanga.xml]: Sānǧā?

File: 5001-6000/LOC5542Sanga.xml [hunk 6]
```diff
@@ -10,7 +10,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title>Sanga</title>
+            <title>Sāngā</title>
```
Comment (thea-m) [inline 5001-6000/LOC5542Sanga.xml]: Sānǧā?

Comment (sologebre) [reply 5001-6000/LOC5542Sanga.xml]: Thank you, I have corrected all!

**Pull request merged**